### ISSUE TITLE
T-Watch 2020 compilation issues

### ISSUE THREAD

**Issue by wizche** [stevelord/AWatch#3]
I'm trying to build this for the 2020 model and noticed that the define is 
`LILYGO_TWATCH_2020_V1`. According to other example sketches it should be `LILYGO_WATCH_2020_V1`.

Despite that I still get a gazillions compilation issues 😄 
Looks like some LVGL API changes, currently (according to TTGO doc) its using v7.0 but the AWatch project seems to use v6 API. Maybe I need to use an older version of the TWatch Library....

Btw... very cool project!

**Comment by chiark** [comment]
Yes, I had the same issue: don't clone the library, download the release version.  Even if you do sort out the defines, you'll need to update from LVGL 6 to 7 as you noticed.  @stevelord has mentioned he will try to look at this, but it makes sense to do it once the updated watch library is released...

**Comment by stevelord** [comment]
Hi @wizche, sorry I was slow getting back to you. I've had some back problems coinciding with a ton of work so I'm not really getting weekends right now. The define was added in through a previous pull request before Lilygo updated their library and created their own. You're right that they've used LVGL V7.0 which breaks a ton of stuff.

It is on the list of things to fix and at the very least update documentation so people are directed to the right library version. Once I've sorted this out I'll close the issue.
